index.html

```diff
@@ -76,188 +91,221 @@ <h1 class="visually-hidden">Интернет-магазин Техномарт</
-              <a class="bestsellers__item-link catalog__item-link" href="#">
-                <h3 class="bestsellers__item-name catalog__item-name">Перфоратор BOSCH BFG 6000</h3>
-                <p class="bestsellers__item-oldprice catalog__item-oldprice">30500 Р.</p>
-                <p class="bestsellers__item-price catalog__item-price">25500 Р.</p>
-              </a>
-              <button class="bestsellers__button bestsellers__button--add-to-cart button button--add-to-cart" type="button">Купить</button>
-              <button class="bestsellers__button bestsellers__button--wishlist button button--wishlist" type="button">В закладки</button>
-            </article>
-          </li>
-          <li class="bestsellers__item catalog__item">
-            <article>
-              <img src="img/item-2.jpg" width="127" height="112" class="bestsellers__item-image catalog__item-image" alt="Перфоратор BOSCH BFG 2000">
-              <a class="bestsellers__item-link catalog__item-link" href="#">
-                <h3 class="bestsellers__item-name catalog__item-name">Перфоратор BOSCH BFG 2000</h3>
-                <p class="bestsellers__item-oldprice catalog__item-oldprice"></p>
-                <p class="bestsellers__item-price catalog__item-price">12500 Р.</p>
-              </a>
-              <button class="bestsellers__button bestsellers__button--add-to-cart button button--add-to-cart" type="button">Купить</button>
-              <button class="bestsellers__button bestsellers__button--wishlist button button--wishlist" type="button">В закладки</button>
-            </article>
-          </li>
-
-        </ul>
+        <div class="wrapper">
 
-      </section>
+          <div class="section__heading-wrapper">
+            <h2 class="section__heading">Популярные товары:</h2>
+            <a class="button button--heading" href="#">Все популярные товары</a>
+          </div>
 
-      <section class="vendors">
+          <ul class="catalog__list catalog__list--bestsellers">
+            <li>
+              <article class="catalog__item">
+                <div class="catalog__buttons">
+                  <button class="catalog__button catalog__button--add-to-cart button" type="button">Купить</button>
+                  <button class="catalog__button catalog__button--wishlist button" type="button">В закладки</button>
+                </div>
+                <img src="img/item-1.jpg" width="184" height="164" class="catalog__item-image" alt="Перфоратор BOSCH BFG 9000">
+                <a class="catalog__item-link" href="#">
+                  <h3 class="catalog__item-name">Перфоратор BOSCH BFG 9000</h3>
+                </a>
+                <p class="catalog__item-oldprice">32500 Р.</p>
+                <p class="catalog__item-price">25500 Р.</p>
+              </article>
+            </li>
+            <li>
+              <article class="catalog__item badge--new">
```

Comment:
по бэм модификатор должен содержать блок и элемент
```suggestion
              <article class="catalog__item catalog__item--new">
```

Reply:
+